doc/Type/independent-routines.pod6

```diff
@@ -688,13 +722,32 @@ Between the C<%> and the format letter, you may specify several
 additional attributes controlling the interpretation of the format. In
 order, these are:
 
-=head3 Format parameter index
+=head3 Format parameter index using the '$' symbol
 
-An explicit format parameter index, such as C<2$>. By default,
-C<sprintf> will format the next unused argument in the list, but this
-allows you to take the arguments out of order:
+An explicit format parameter integer index (ranging from 1 to N args)
+before the directive, such as C<2$>. By default, C<sprintf> will
```

Comment:
_that goes_ before the directive? Also, show the directive in the example.

Reply:
Good catch, will fix.